src/application/components/Explorer/Explorer.tsx

```diff
@@ -50,7 +53,7 @@ function executeOperation({
   isSubscription?: boolean;
   clientId: string;
 }) {
-  return new Observable<QueryResult>((observer) => {
+  return new Observable<ExplorerResponse>((observer) => {
```

Comment:
I reworked how explorer requests/responses work since v3/v4 returns different result types for the different operations. Now instead of trying to shoehorn both clients into an abstract type which is then changed to an `ExplorerResponse` here, the content script just returns the explorer response directly.